Storybook!

b0g3r opened the pull request:
Closes #11

Commit:
Add storybook

Commit:
Fix storybook issues

Commit:
Add comments and remove bad files

b0g3r commented:
blocked by #10

Gilbord commented on src/components/PublicationListItem.vue:
```diff
@@ -1,27 +1,46 @@
+<docs>
+Hey **hello**
```
Я так понимаю, компонент этот будет доделываться и в последствии будет лучше написана дока?

b0g3r replied:
Да. Сейчас этот компонент для того чтобы от этого мёрдж-реквеста можно было работать дальше — видно как создавать компоненты, видно как создавать истории

Gilbord commented on src/stories/PublicationListItem.js:
```diff
@@ -0,0 +1,22 @@
+
```
Насколько тут нужна пустая строка?

Gilbord commented on src/stories/PublicationListItem.js:
```diff
@@ -0,0 +1,22 @@
+        :text="text"
+      ></publication-list-item>`,
+    }
+  });
```
Отсутствует новая строка в конце файла.

Gilbord commented on vue.config.js:
```diff
@@ -0,0 +1,14 @@
+        ]
+      }
+    }
+  }
```
Отсутствует новая строка в конце файла.

Commit:
Update storybook to five version :tada:

For more info see: https://github.com/storybooks/storybook/issues/4947#issuecomment-468337583

Commit:
Fix issue with missed emotion, just re-lock deps

See more: https://github.com/storybooks/storybook/issues/4947#issuecomment-468374847

Commit:
Add build stage for storybook

Commit:
Fix linter issue

Commit:
Add missed new line

teners reviewed (approved):
гуддо
